lib/diagnostic.js

```diff
@@ -38,19 +50,30 @@ let owner = null;
 // Methods: vocalize
 
 // Replace null with your answer
-let pet = null;
```

Comment:
We're looking for something with the following form.
```javascript
const pet = {
  name: 'some name',
  species: 'some species',
  breed: 'some breed',
  vocalization: 'some sound'
};
```
What you gave was a nested object with ```pet.Attributes``` as the object that we're looking for.  An error is also thrown because ```vocalize``` doesn't exist.